docs/development/vsphere-dev.md

```diff
@@ -102,7 +102,7 @@ Use .build/dist/linux/amd64/kops if working on a linux machine, instead of mac.
 **Notes**
 
 1. ```clustername``` should end with **skydns.local**. Example: ```kubernetes.cluster.skydns.local```.
-2. ```zones``` should end with ```a```. Example: ```us-west-2a``` or as the above command sets ```--zones=${AWS_REGION}a```.
+2. For ```zones``` any string will do, for now. It's only getting used for the construction of names of various entities. But it's a mandatory argument.
```

Comment:
if any strings will do, why not have a default ? Or, are you saying it will be mandatory in the future when the string will have meaning ?